core/__init__.py

```diff
@@ -220,8 +239,8 @@ def setup(modules: Union[object, ModuleType], render: Union[ModuleType, Dict] =
     # noinspection PyUnresolvedReferences
     import skeletons  # This import is not used here but _must_ remain to ensure that the
     # application's data models are explicitly imported at some place!
-    assert utils.projectID in conf["viur.validApplicationIDs"], \
-        "Refusing to start, applicationID %s is not in conf['viur.validApplicationIDs']" % utils.projectID
+    assert conf["viur.instance.project_id"] in conf["viur.validApplicationIDs"], \
+        f"""Refusing to start, {conf["viur.instance.project_id"]=} is not in {conf["viur.validApplicationIDs"]=}"""
```

Comment:
As already described in https://github.com/viur-framework/viur-core/pull/507#discussion_r970617891 please replace these asserts with exceptions.

```suggestion
    if conf["viur.instance.project_id"] not in conf["viur.validApplicationIDs"]:
        raise RuntimeError(f"""Refusing to start, {conf["viur.instance.project_id"]=} is not in {conf["viur.validApplicationIDs"]=}""")
```